pkg/registry/core/pod/strategy.go

```diff
@@ -380,6 +380,14 @@ func streamParams(params url.Values, opts runtime.Object) error {
 		if opts.TTY {
 			params.Add(api.ExecTTYParam, "1")
 		}
+	case *api.PodPortForwardOptions:
+		if len(opts.Ports) > 0 {
```

Comment:
does it make sense to throw and error here if `Ports` is empty? We require at least one port in the server.